TeachingRecordSystem/tests/TeachingRecordSystem.SupportUi.Tests/PageTests/Persons/PersonDetail/InductionTests.cs

```diff
@@ -0,0 +1,232 @@
+    {
+        // Arrange
+        var person = await TestData.CreatePersonAsync(
+            builder => builder
+                .WithQts()
+                .WithInductionStatus(builder => builder
+                    .WithStatus(setInductionStatus)
+                    )
+                );
+
+        var request = new HttpRequestMessage(HttpMethod.Get, $"/persons/{person.ContactId}/induction");
+
+        // Act
+        var response = await HttpClient.SendAsync(request);
+
+        // Assert
+        var doc = await AssertEx.HtmlResponseAsync(response);
+        var inductionStatus = doc.GetElementByTestId("induction-status");
+        Assert.Contains(StatusStrings[setInductionStatus], inductionStatus!.TextContent);
+        Assert.Null(doc.GetElementByTestId("induction-start-date"));
+        Assert.Null(doc.GetElementByTestId("induction-end-date"));
+        Assert.NotNull(doc.GetAllElementsByTestId("induction-backlink"));
+    }
+
+    [Theory]
+    [InlineData(InductionStatus.Exempt)]
+    public async Task Get_WithPersonIdForPersonWithInductionStatusExempt_DisplaysExpected(InductionStatus setInductionStatus)
+    {
+        // Arrange
+        var person = await TestData.CreatePersonAsync(
+            builder => builder
+                .WithQts()
+                .WithInductionStatus(builder => builder
+                    .WithStatus(setInductionStatus)
+                    .WithExemptionReasons(InductionExemptionReasons.SomethingMadeUpForNow)
+                    )
+                );
+
+        var request = new HttpRequestMessage(HttpMethod.Get, $"/persons/{person.ContactId}/induction");
+
+        // Act
+        var response = await HttpClient.SendAsync(request);
+
+        // Assert
+        var doc = await AssertEx.HtmlResponseAsync(response);
+        var inductionStatus = doc.GetElementByTestId("induction-status");
+        Assert.Contains(StatusStrings[setInductionStatus], inductionStatus!.TextContent);
+        var exemptionReason = doc.GetElementByTestId("induction-exemption-reasons")!.Children[1].TextContent;
+        Assert.Contains("SomethingMadeUpForNow", exemptionReason); // CML TODO - needs proper mapping to string
+        Assert.Null(doc.GetElementByTestId("induction-start-date"));
+        Assert.Null(doc.GetElementByTestId("induction-end-date"));
+        Assert.NotNull(doc.GetAllElementsByTestId("induction-backlink"));
+    }
+
+    [Theory]
+    [InlineData(InductionStatus.InProgress)]
+    [InlineData(InductionStatus.Passed)]
+    [InlineData(InductionStatus.Failed)]
+    [InlineData(InductionStatus.FailedInWales)]
+    public async Task Get_WithPersonIdForPersonWithInductionStatusRequiringStartDate_DisplaysExpectedContent(InductionStatus setInductionStatus)
+    {
+        // Arrange
+        //var expectedWarning = "To change a teacher's induction status ";
+        var setStartDate = DateOnly.FromDateTime(DateTime.Now.AddMonths(-1));
```

Comment:
We use an abstraction for getting the current date so we can fake it in tests:
```suggestion
        var setStartDate = Clock.Today.AddMonths(-1);
```